src/tests/tests/firstparties.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+  let tco = 'http://t.co/beach-detour/';
+  let destination = 'https://the.beach/';
+
+  document.querySelectorAllBefore = document.querySelectorAll;
```

Comment:
We should probably make whatever hacks we need to make in `before` or `beforeEach` properties of the [QUnit module](https://api.qunitjs.com/QUnit/module). This way the stubbing doesn't happen for all tests, just the ones it should happen for.